### DISCUSSION TITLE
Virtualized combobox with infinity data loading

### DISCUSSION THREAD

**Discussion by PrinceLoren** [shadcn-ui/ui#2074] [Ideas]
Demo -> https://shadcn-combobox-virtualized-infinity.vercel.app/
GITHUB  -> https://github.com/PrinceLoren/shadcn-combobox-virtualized-infinity

I found an example of how it's done with normal virtualization without data subloading

btw, would really like to see this in shadcn's realization @shadcn

**Comment by ViniciusSCG** [comment]
You can simplify, just pass ScrollArea Component inside CommandGroup

like

`<CommandGroup>
              <ScrollArea className="h-24">
                {data?.map((item) => (
                  <CommandItem
                    value={item.label || ""}
                    key={item.value}
                    onSelect={() => {
                      form.setValue(field.name, item.value);
                    }}
                  >
                    <Check
                      className={cn(
                        "mr-2 h-4 w-4",
                        item.value === field.value ? "opacity-100" : "opacity-0"
                      )}
                    />
                    {item.label}
                  </CommandItem>
                ))}
              </ScrollArea>
            </CommandGroup>`
